<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Protect_Process_Ring3Class</class>
 <widget class="QMainWindow" name="Protect_Process_Ring3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protect_Process_Ring3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="_Protect_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Protect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_Clear_pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Protect_Process_Ring3.qrc"/>
 </resources>
 <connections/>
</ui>
